kafka-iceberg/apache-flink/README.md

```diff
@@ -0,0 +1,327 @@
+# Streaming data from Kafka to Iceberg with Apache Flink
+
+`@rmoff 2024-07-02`
```

Comment:
Not generally opposed to it, but I don't think we've called out individual authorship for these examples before? What would be the implications of it, like are you going to be the main maintainer for this one? Worth clarifying IMO.

Reply:
removed.